src/internal/utils/tests/sort.spec.ts

```diff
@@ -365,6 +365,25 @@ describe("createSorts", () => {
         };
         expect(createSorts("table", emptyAfm, emptyResultSpec, visualizationProperties)).toEqual([]);
     });
+
+    it("should sort by measure size if chart is geo Pushpin", () => {
+        const afm: AFM.IAfm = {
+            measures: [M1],
+            attributes: [A1, A2],
+        };
+        const resultSpec: AFM.IResultSpec = {
+            dimensions: [
+                { itemIdentifiers: ["measureGroup"] },
+                {
+                    itemIdentifiers: ["a1", "a2"],
+                },
+            ],
+        };
+        const expectedSort: AFM.SortItem[] = [SORTED_BY_M1];
+        expect(createSorts("pushpin", afm, resultSpec, emptyVisualizationProperties, false, false)).toEqual(
```

Comment:
`canSortStackTotalValue` and `enableSortingByTotalGroup` are false by default so we can ommit its values